README.md

```diff
@@ -1,2 +1,353 @@
+
+</p>
+</details>
+
+### Selecting a Model
+
+You can change the model. The default model is Res-UNet.
+**Available options for 'model' are: AttentionU-Net,UNETR**.
+
+
+### Selecting a Dataset
+
+In the same way as the model, also the dataset can be changed. The default dataset is Lits.
+**Available options for 'dataset' are: Lits, AbdomenCT**.
+
+### Changing Hyperparmeters
+
+Our Benchmarking System allowed us to systematically fine-tune hyperparameters and explore various loss functions, including region-based losses like Tversky, Dice and Focal Loss as well as compound losses like Dice Cross Entropy. We also evaluated the performance of both Adam and SGD optimizers, utilizing their respective update strategies. Furthermore, we incorporated schedulers such as AdaptiveLR and Plateau, which dynamically adjust the learning rate based on the training progress.
+
+### Logging and Checkpointing
+#### [Start using Tensorboard](https://pytorch.org/tutorials/recipes/recipes/tensorboard_with_pytorch.html#run-tensorboard)
+
+The logging structure of the output folder is depicted below.
+The ``LOGDIR=<some.folder.dir>`` argument defines the logging folder (*"logs/"* by default).
+For a better overview, experiments can also be named by ``name="my_experiment"`` ("basemodel"
+by default).
+The parameters which are logged are: Hyperparameters (like epochs, batch_size, initial lr,...), 
+metrics, loss (train+val), time (train+val) and learning rate.
+
+````
+LOGDIR                       # logs/ by default
+    └──checkpoints/          # If checkpointing is enabled this contains the best and the last epochs checkpoint
+    ├── logs/                # Console logging file
+    └── events/              # Tensorboard logs      
+````
+
+Since [Tensorboard](https://pytorch.org/tutorials/recipes/recipes/tensorboard_with_pytorch.html#run-tensorboard)
+is used for logging, you can view the logged results by the following command.
+Tensorboard will include all logs (*.tfevents.* files) found in any subdirectory of *--logdir*.
+This means by the level of the selected *--logdir* you can define which experiments (runs) should be
+included into the tensorboard session.
+
+````shell
+tensorboard --logdir=<some.dir>
+````
+
+#### [Start using ClearML](https://clear.ml/docs/latest/docs/getting_started/ds/ds_first_steps) 
+
+The ClearML run-time components:
+
+* The ClearML Python Package - for integrating ClearML into scripts by adding just two lines of code, and optionally extending experiments and other workflows with ClearML's powerful and versatile set of classes and methods.
+* The ClearML Server - for storing experiment, model, and workflow data; supporting the Web UI experiment manager and MLOps automation for reproducibility and tuning. It is available as a hosted service and open source to deploy ClearML Server.
+
+<img src="https://i.imgur.com/purP0Rs.jpeg" width="100%" alt="clearml-architecture">
```

Comment:
The image resolution is very low; Please increase the resolution and you can upload it to this repo, maybe under resources dir